### PR TITLE
Proposal to change max vp

### PR DESCRIPTION
**Pull request by BrandonWeng**
Tested locally:

Before
```
seid q staking params
bond_denom: usei
historical_entries: 10000
max_entries: 7
max_validators: 35
max_voting_power_enforcement_threshold: "1000000"
max_voting_power_ratio: "0.200000000000000000"
min_commission_rate: "0.050000000000000000"
unbonding_time: 1814400s
```

After
```
seid q staking params
bond_denom: usei
historical_entries: 10000
max_entries: 7
max_validators: 35
max_voting_power_enforcement_threshold: "1000"
max_voting_power_ratio: "0.150000000000000000"
min_commission_rate: "0.050000000000000000"
unbonding_time: 1814400s
```

### CODE REVIEW

File: atlantic-2/proposals/03-03-2022-decrease-vp-threshold.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+    "title": "Change Max Voting PowerEnforcement",
+    "description": "Decrese threshold to start enforcing threshold soon",
+    "changes": [
+      {
+        "subspace": "staking",
+        "key": "MaxVotingPower",
```
Comment (udpatil) [inline atlantic-2/proposals/03-03-2022-decrease-vp-threshold.json:7]: is this key supposed to be MaxVotingPowerRatio?

Comment (BrandonWeng) [reply atlantic-2/proposals/03-03-2022-decrease-vp-threshold.json:7]: no... https://github.com/sei-protocol/sei-cosmos/blob/main/x/staking/types/params.go#L86

Comment (udpatil) [reply atlantic-2/proposals/03-03-2022-decrease-vp-threshold.json:7]: ok sounds good, just verifying since in the queried params it outputs `ratio`

**Review by udpatil — APPROVED** [review]

File: atlantic-2/proposals/03-03-2022-decrease-vp-threshold.json (hunk 1 above)
Comment (BrandonWeng) [reply atlantic-2/proposals/03-03-2022-decrease-vp-threshold.json:7]: yeah, i dont like it tbh. i thought it would be MaxPowerRatio at first too

**Pull request merged**